flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerServices.java

```diff
@@ -329,20 +328,22 @@ private static TaskSlotTable createTaskSlotTable(
 	 * directories (not files), and are writable.
 	 *
 	 * @param tmpDirs The array of directory paths to check.
-	 * @throws IOException Thrown if any of the directories does not exist and cannot be created or is not writable
-	 *                     or is a file, rather than a directory.
+	 * @throws IOException Thrown if any of the directories does not exist and cannot be created or is not writable or
```

Comment:
unrelated change and others in this function

Reply:
Don't know what happened with this commit. So many format changes that I'm not aware of. Probably some triggered some IDE function by accident.